.env-example

```diff
@@ -1,8 +1,8 @@
-NEXMO_API_KEY=NEXMO_API_KEY
-NEXMO_API_SECRET=NEXMO_API_SECRET
-NEXMO_APPLICATION_ID=NEXMO_APPLICATION_ID
-NEXMO_PRIVATE_KEY_PATH=NEXMO_PRIVATE_KEY_PATH
-NEXMO_NUMBER=YOUR_NEXMO_NUMBER
+VONAGE_API_KEY=VONAGE_API_KEY
+VONAGE_API_SECRET=VONAGE_API_SECRET
+VONAGE_APPLICATION_ID=VONAGE_APPLICATION_ID
+VONAGE_PRIVATE_KEY_PATH=VONAGE_PRIVATE_KEY_PATH
+VONAGE_NUMBER=YOUR_VONAGE_NUMBER
 TO_NUMBER=NUMBER_TO_CALL
```

Comment:
Lots of variables missing from the .env file

```NUMBER_SEARCH_CRITERIA=
NUMBER_SEARCH_PATTERN=
COUNTRY_CODE=US
VONAGE_NUMBER_FEATURES=
SMS_CALLBACK_URL=
VOICE_CALLBACK_TYPE=
VOICE_CALLBACK_VALUE=
VOICE_STATUS_URL=
INSIGHT_NUMBER=
ASYNC_CALLBACK_URL=
```

Reply:
**INSIGHT_NUMBER** is at the bottom of the .env-example file but I can definitely add the others